src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -134,6 +134,7 @@ static UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_t, uct_md_h md, uct_worker_h worker
                                                self->tx.bb_max / 4);
 
     status = uct_rc_mlx5_iface_common_init(&self->mlx5_common, &self->super, &config->super);
+    uct_ib_iface_set_max_iov(&self->super.super, self->super.config.tx_min_sge);
```

Comment:
@yosefe I don't understand `sizeof(mlx5_wqe_av_seg)` for DC. Did you mean `UCT_IB_MLX5_AV_FULL_SIZE`?